src/games/strategy/engine/framework/GameRunner.java

```diff
@@ -139,6 +142,16 @@ public static void main(final String[] args) {
     ErrorConsole.getConsole();
     // do after we handle command line args
     checkForMemoryXMX();
+    Toolkit.getDefaultToolkit().getSystemEventQueue().push(new EventQueue() {
```

Comment:
Could use clarification how this change connects with the line removal in ClientLogger. Otherwise there is some magic going on with the toolkit event queue. If we can avoid it, would be good, otherwise a quick comment about how the magic works would be helpful